### PR TITLE
Continuous deployment

### LINKED ISSUE #107: Automate Firefox release

### LINKED ISSUE #108: Automate Chrome release

### PR DESCRIPTION
**Pull request by ttoino**
Closes #107.
Closes #108.

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        with:
+          name: firefox
+      - name: Sign add-on
+        run: npx web-ext sign --api-key $API_KEY --api-secret $API_SECRET
+      - name: Upload signed add-on
+        uses: actions/upload-artifact@v2
+        with:
+          path: web-ext-artifacts
+          name: firefox-signed
+
```
Comment (LuisDuarte1) [inline .github/workflows/cd.yml:85]: Should we trigger a `nitsig-website` action to update itself, here?

**Review by LuisDuarte1 — COMMENTED** [review]
Good job! It seems okay, just have some comments below:

File: .github/workflows/cd.yml (hunk 1 above)
Comment (ttoino) [reply .github/workflows/cd.yml:85]: We should find a way to update the website automatically, I think triggering an action there is possible like you said. I'll investigate.

**Review by LuisDuarte1 — APPROVED** [review]
LGTM

**Review by pedrosilva17 — APPROVED** [review]
👌 impeccable

**Pull request merged**